src/layout/Footer.jsx

```diff
@@ -1,7 +1,95 @@
 import React from "react";
+import { Link as GatsbyLink } from "gatsby";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import { Facebook, Twitter, RssFeed } from "@material-ui/icons";
+import Grid from "@material-ui/core/Grid";
+
+/* Defines CSS classes to use for all instances of this component. This is
+  actually a custom React hook. */
+const useStyles = makeStyles((theme) => ({
+  footerLink: {
+    color: theme.palette.text.secondary,
+  },
+}));
 
 const Footer = ({ title }) => {
-  return <>&copy; {title} 2020</>;
+  const classes = useStyles(); //consumes ThemeProvider 
+  return (
+    <>
+      <hr />
+      <Typography
+        variant="h5"
+        color="textSecondary"
+        align="center"
+        noWrap
+      >
+        <Link
+          to="/"
+          component={GatsbyLink}
+          color="inherit"
+        >
+          {title}
+        </Link>
+      </Typography>
+      <Grid 
+        container 
+        direction="row"
+        justify="center" 
+        alignItems="center"
+        spacing={2}
+      >
+        <Grid item>
+          <Link
+            href="https://www.facebook.com/huntercsblog"
+          >
+            <Facebook />
```

Comment:
Is this intentionally left blue to mimic facebook colors? Will the other icons be blue as well? Will they be colors more loyal to their sites?

Reply:
Yes I left it the facebook color on purpose. I plan to color the other icons when I add the proper links. Light blue for Twitter and Orange for RSS. Since they're small, color clashes shouldn't be noticeable